packages/system-upgrade-controller/package.yaml

```diff
@@ -1,2 +1,4 @@
-url: local
-version: 105.0.1
+auto: true
+url: https://github.com/rancher/system-upgrade-controller.git
+chartRepoBranch: rancher-charts-release/v2.10
```

Comment:
Why is the [rancher-charts-release/v2.10](https://github.com/rancher/system-upgrade-controller/tree/rancher-charts-release/v2.10) branch empty? I expect the see the templates in the `charts` subdirectory there.

Reply:
We've PR there https://github.com/rancher/system-upgrade-controller/pull/356 to add charts.